### PR TITLE
Get Old Sidebar working again

### PR DESCRIPTION
**Pull request by estufar**
Tumblr changing the width of the sidebar reminded me I really needed to get to fixing this, so I did that. This will end up fixing #1034 (finally)

It won't be compatible with the current version of the sidebar width tweak though. This changes the widths of the sidebar and container to the older widths (I think they're accurate at least), whereas the tweak adjusts the padding, which didn't quite work with this. If we do want to change the tweaks version to use the same styles to adjust the width, I think it'll work with the standard sidebar too.

### CODE REVIEW

**Comment by nightpool** [comment]
I would prefer to have old sidebar use the new larger widths by default, with only the tweak option changing the column sizes. That way we remain consistent as far as user expectations go.

File: Extensions/estufars_sidebar_fix.js [hunk 1]
```diff
@@ -54,20 +54,35 @@ XKit.extensions.estufars_sidebar_fix = new Object({
...
 			account.style.display = "none";
 			// wait and then let tumblr know the menu is no longer active
 			window.setTimeout(function() {
 				document.querySelector(".tab_nav_account.active").click();
-			}, 500);
-		}, 250);
+			}, 250);
+		}
+		
+		if($(".popover--account-popover").length == 0) {
```
Comment (nightpool) [inline Extensions/estufars_sidebar_fix.js]: should be `!$(".popover--acount-popover").length`

**Comment by estufar** [comment]
Sorry about that mess of commits, #1076 was merged between mine and seems to be messing with being able to squash them

**Comment by a-mercurial-one** [comment]
Hi, I have no ability to participate in this work but I've been, first avidly, as it appeared that this fix was progressing, and recently only at times checking, but I'd like to chime in that I find this to be one of the most important tumblr extensions out there and really hope to see it functional again.

**Comment by nightpool** [comment]
Tested, works locally, LGTM. 

Sorry about the delay in getting this merged

@homu r+

**Comment by nightpool** [comment]
closed in favor of #1302

(question—is the "allow maintainers to edit this PR" box checked? I don't think I've ever had that work successfully, but this PR might have been created before that was a thing....)

**Pull request closed without merging**